src/mixins/nr-affiliation-mixin.ts

```diff
@@ -89,11 +89,11 @@ export class NrAffiliationMixin extends Mixins(CommonMixin) {
           return
         }
 
-        this.setAffiliationErrorModalValue(NrAffiliationErrors.UNABLE_TO_START_REGISTRATION)
+        this.setAffiliationErrorModalValue(NrAffiliationErrors.ASSOCIATED_OTHER_ACCOUNT)
         throw Error('Unable to find existing affiliation')
       }
 
-      this.setAffiliationErrorModalValue(NrAffiliationErrors.ASSOCIATED_OTHER_ACCOUNT)
+      this.setAffiliationErrorModalValue(NrAffiliationErrors.UNABLE_TO_START_REGISTRATION)
```

Comment:
If we get to this point then we were unable to create a new affiliation and the NR is not already affiliated.

This is a "technical problem" so use this version of the affiliation error dialog.